### ISSUE TITLE
can not customize sass varibale '$color-select-border-default' of Select component using semi webpack theme plugin

### ISSUE THREAD

**Issue by summerstream** [DouyinFE/semi-design#375]
### Which Component 出现bug的组件
  - Select

### semi-ui version
  - latest
    
### Expected result 期望的结果是什么
  - can use semi theme webpack plugin to customize the value of $color-select-border-default

### Actual result 实际的结果是什么
  - custom theme do not work, the value of $color-select-border-default is always the default 'transparent'

### Steps to reproduce 复现步骤
  - 1 use below semi webpack config below:
    ```
      new SemiPlugin({
        theme: {
          name: 'xx-theme',
        },
        variables: {
          "$color-select-border-default": "red",
        },
      })
    ```
    - 

### Reproducible code 复现代码
```
class Demo extends React.component {

    render() {
        return (
            <Select defaultValue='abc' disabled style={{ width: 120 }}>
                <Select.Option value='abc'>抖音</Select.Option>
                <Select.Option value='hotsoon'>火山</Select.Option>
            </Select>
        )
    }

}

```

### Additional information 补充说明
- 遇到这个bug的业务场景、上下文、或者你的需求场景

**Comment by pointhalo** [comment]
@wuhw 确认下，以variables方式来覆盖token是不是已经不再推荐的做法？

**Comment by pointhalo** [comment]
fix 2.2.0-beta.0
